models/twilio_number_overview.sql

```diff
@@ -1,15 +1,15 @@
-with message_enhanced as (
+with message_enhanced as ( 
 
     select *
     from {{ ref('twilio_message_enhanced')}}
 )
 
 select
-    phone_number,
-    count(case
-        when direction like '%outbound%'
-        then message_id end)
-        as total_outbound_messages,
+    phone_number, 
```

Comment:
Didn't we also want to include country code in this model?

Reply:
See earlier note

Reply:
Thanks! We can circle back on this